src/Forms/FieldList.php

```diff
@@ -91,9 +85,9 @@ public function flattenFields()
      * Return a sequential set of all fields that have data.  This excludes wrapper composite fields
      * as well as heading / help text fields.
      *
-     * @return FormField[]
+     * @return array<FormField>|null
      */
-    public function dataFields()
+    public function dataFields(): ?array
```

Comment:
Should just return `array` due to the change in https://github.com/silverstripe/silverstripe-framework/pull/11235#discussion_r1599180952

Reply:
Done